extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -132,7 +133,8 @@ export class DoubleclickA4aEligibility {
           'https://github.com/ampproject/amphtml/issues/11834 ' +
           'for more information');
     const usdrd = 'useSameDomainRenderingUntilDeprecated';
-    const hasUSDRD = usdrd in element.dataset || element.hasAttribute(usdrd);
+    const hasUSDRD = (tryParseJson(element.getAttribute('json')) || {})[usdrd]
```

Comment:
Actually let's reverse the items in this or as usdrd in element.dataset is faster to execute

Reply:
done